src/bp/lang-server/service/index.ts

```diff
@@ -45,75 +54,172 @@ export default class LanguageService {
     return this._ready
   }
 
-  listModels(): AvailableModel[] {
-    return _.values(this._models)
+  listFastTextModels(): AvailableModel[] {
+    const allModels = _.values(this._models)
+    return allModels.map(m => m.fastTextModel)
   }
```

Comment:
listFastTextModels = (): AvailableModel[] =>
    _.values(this._models).map(m => m.fastTextModel)

Reply:
Not sure about this as I am not too familiar with our code style yet.